app/assets/javascripts/modules/image_x_viewer.js

```diff
@@ -253,7 +253,10 @@
     var _setupButtonListeners = function() {
       $embedHeader.find('[data-sul-view-fullscreen="fullscreen"]')
         .on('click', function() {
+          var worldRestriction = $('[data-world-restriction]').data('world-restriction');
```

Comment:
I think you can get away with removing this. This should already be stored in an object `dataAttributes.worldRestriction`.